sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client.hpp

```diff
@@ -11,8 +11,10 @@
 #include <azure/core/credentials.hpp>
 #include <azure/core/http/http.hpp>
 #include <azure/core/response.hpp>
+
```

Comment:
```suggestion
```

Reply:
intentionally space. Above is headers from azure core. Below is from KeyVault Common